docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -17,17 +17,18 @@
         wait_for_timeout: " {{ wait_for_timeout | default(400) }}"
         boot_wait_time: " {{ boot_wait_time | default(180) }}"
 
-    - set_fact:
-        helm_chart_overrides: "{{ deployment_manager_overrides }}"
-      when: deployment_manager_overrides is defined
-
     - set_fact:
         deploy_config: "{{ deployment_config }}"
         wait_for_dm_unlock: " {{ wait_for_dm_unlock| default(5) }}"
       when: deployment_config is defined
 
     - set_fact:
         manager_app_chart: "{{ manager_app_chart | default('/usr/local/share/applications/helm/deployment-manager-*.tgz') }}"
+      when: manager_app_chart is defined
+
+    - set_fact:
+        application_helm_chart_dir: "{{ application_helm_chart_dir | default('/usr/local/share/applications/helm/') }}"
```

Comment:
TBH, I don't know the purpose why we need an additional variable "application_helm_chart_dir", we can simply override the variable manager_app_chart. 
And it is dangerous to find a patterns: 'deployment-manager*.tgz', there may be more than one file has this pattern. And the register app_helm_chart was never used